modules/acquia_cms_toolbar/acquia_cms_toolbar.libraries.yml

```diff
@@ -1,8 +1,3 @@
-styling:
-  version: VERSION
-  css:
-    theme:
-      css/acquia_cms_toolbar.css: {}
```

Comment:
This is required for the below:
- Environment specific background color to environment variable.
- Admin menu styles.

Reply:
As per discussion in DSM included styles from Environment specific background color to environment variable but admin menu styles are removed.